### DISCUSSION TITLE
Alters and multibox

### DISCUSSION THREAD

**Discussion by Gorthu** [chromiecraft/chromiecraft#3214] [General]
Hello. I have a question about how to level up secondary characters. If multibox is allowed, I can level up secondary characters by entering solo dungeons. (without using the dungeon finder, entering through door). Of course, this can only be done up to a certain level, after that you continue in the regular way, doing fetch or rdf. This is a practice that is carried out on all servers with multiple accounts, but since it does not seem to me that it is well defined in the rules here, I prefer to ask, before receiving one of those bans without warning, and with tests that I do not know if they exist , because they don't teach them. I don't want to be accused of being a cheat for raising the initial levels quickly, and that is done in a normal way in other places. It is not a new idea that I am bringing, but as I see players who are doing it, I just want to be sure

**Comment by 55Honey** [comment]
Hi Gorthu, thanks for reaching out.
"Boosting" characters through dungeons is a common practice and legal part of the game. We do not have rules against this.
